.github/workflows/luacheck.yml

```diff
@@ -11,3 +11,11 @@ jobs:
         uses: actions/checkout@v3
       - name: Luacheck
         uses: lunarmodules/luacheck@v0
+
```

Comment:
Lets move this to it's own workflow, it shouldn't be part of the `luacheck` workflow it should be a standalone one (you can copy the rest of Luacheck workflow as boilerplate to start off with).